ext/scimv2/logic/pom.xml

```diff
@@ -61,7 +61,19 @@ under the License.
       <groupId>org.antlr</groupId>
       <artifactId>antlr4-runtime</artifactId>
     </dependency>
-    
+
```

Comment:
Only `mockito-junit-jupiter`  should be needed.
Please remove `mockito-core`.
Also, remove the blank line after the `mockito-junit-jupiter` block